src/topology/algebra/uniform_convergence.lean

```diff
@@ -18,29 +19,39 @@ space of continuous linear maps between two topological vector spaces.
 * `uniform_convergence.uniform_group` : if `G` is a uniform group, then the uniform structure of
   uniform convergence makes `α → G` a uniform group
 * `uniform_convergence_on.uniform_group` : if `G` is a uniform group, then the uniform structure of
-  `𝔖`-convergence, for any `𝔖 : set (set α)`, makes `α → G` a uniform group
+  `𝔖`-convergence, for any `𝔖 : set (set α)`, makes `α → G` an uniform group.
+* `uniform_convergence_on.has_continuous_smul_of_image_bounded` : let `E` be a TVS,
+  `𝔖 : set (set α)` and `H` a submodule of `α → E`. If the image of any `S ∈ 𝔖` by any `u ∈ H` is
+  bounded (in the sense of `bornology.is_vonN_bounded`), then `H`, equipped with the topology of
+  `𝔖`-convergence, is a TVS.
 
 ## TODO
 
-* Let `E` be a TVS, `𝔖 : set (set α)` and `H` a submodule of `α → E`. If the image of any `S ∈ 𝔖`
-  by any `u ∈ H` is bounded (in the sense of `bornology.is_vonN_bounded`), then `H`, equipped with
-  the topology of `𝔖`-convergence, is a TVS.
+* `uniform_convergence_on.has_continuous_smul_of_image_bounded` unnecessarily asks for `𝔖` to be
+  nonempty and directed. This will be easy to solve once we know that replacing `𝔖` by its
+  ***noncovering*** bornology (i.e ***not*** what `bornology` currently refers to in mathlib)
+  doesn't change the topology.
 
 ## References
 
 * [N. Bourbaki, *General Topology, Chapter X*][bourbaki1966]
+* [N. Bourbaki, *Topological Vector Spaces*][bourbaki1987]
 
 ## Tags
 
 uniform convergence, strong dual
 
 -/
 
+open filter
+open_locale topological_space pointwise
+
 section group
 
 variables {α G : Type*} [group G] [uniform_space G] [uniform_group G] {𝔖 : set $ set α}
```

Comment:
```suggestion
variables {α G ι : Type*} [group G] [uniform_space G] [uniform_group G] {𝔖 : set $ set α}
```
and removing it later on (I really don't like having types definitions in lemmas).